AbijahKaj/wakala-interface
# Updated contract methods to use Contract Kit (#265)

* Added Redux and AsyncStorage

* Update App.js

* Update index.js

* Update index.js

* Adding icon and Splash Screen

* Adding working PIN library

* Ejected from Expo to support AsyncStorage

* Released the #1 APK

* Changing the splash screen display mode

* Tested and fixed the iOS version

* Update app-release.apk

* Hide the onboarding section when the user already went through it

All new screens should now be added under src/screens/index.js

* Revert "Merge branch 'dev' of https://github.com/WakalaDAO/wakala-interface into dev"

This reverts commit b2e1ec7f8a29f65ba8b0dd1405eaaf0ea21620b7, reversing
changes made to 3d9aa3ea922d527c438f8c255d529ffadbb7f36d.

* Update app-release.apk

* Update VerifyCeloCodes.js

* Added logout support (not fully)

* Started Integrating Magic and Contracts within Redux

* Implemented the logout process

* Revert "Merge branch 'dev' of https://github.com/WakalaDAO/wakala-interface into dev"

This reverts commit ba970d00d8fae69be700281e94f27246d69b606c, reversing
changes made to 9ac76713a6dcc8c7d915a453e145de47e5e1935b.

* Update app-release.apk

* Finished integrating with the Smart Contract

We now have to use those methods in the app

* Added support for balance and user's public address

* Withdrawal process

* Create splash.png

* Updated the Splash Screen

* Update app-release.apk

* Added animation

* Started integrating ERC20

* Updated contract methods to use Contract Kit

* Update ConfirmFunds.js

* Update ConfirmFunds.js

* Update ContractMethods.js

* Update index.js

* Update index.js

* Removed Lottie from package.json
